src/main/java/seedu/partyplanet/logic/parser/ListCommandParser.java

```diff
@@ -124,17 +148,21 @@ public ListCommand parse(String args) throws ParseException {
     private Comparator<Person> getSortOrder(ArgumentMultimap argMap) throws ParseException {
         Optional<String> sortType = argMap.getValue(PREFIX_SORT);
         if (sortType.isEmpty()) {
+            stringSort += "Sorted name ";
             return SORT_NAME; // default
         } else {
             switch (sortType.get().toLowerCase()) {
             case "n": // fallthrough
             case "name":
+                stringSort += "Sorted name ";
                 return SORT_NAME;
             case "b": // fallthrough
             case "birthday":
+                stringSort += "Sorted birthday ";
```

Comment:
```suggestion
                stringSort += "Sorted birthdays ";
```

and in "chronological order" instead of "ascending" maybe?